### PR TITLE
RUN-2974 default backgroundColor to #0FFF when transparent is true

### PR DESCRIPTION
**Pull request by HarsimranSingh**
Sister PR to https://github.com/openfin/runtime/pull/580 (already merged)

### CODE REVIEW

**Comment by nicholasdgoodman** [comment]
What is consuming these options that supports \#ARGB or \#AARRGGBB style color codes?
CSS as a spec doesn't AFAIK.

**Comment by nicholasdgoodman** [comment]
Correction, the CSS 4 draft includes hex notation for alpha channel, but these are in RGBA order, *not* ARGB:

https://drafts.csswg.org/css-color/#hex-notation

**Comment by nicholasdgoodman** [comment]
(One-man convo here) Yikes! It's Electron who's done it this way:
https://github.com/electron/electron/pull/4201

File: src/browser/convert_options.js [hunk 1]
```diff
@@ -291,6 +291,11 @@ module.exports = {
             newOptions.customRequestHeaders = options.customRequestHeaders;
         }
 
+        // implicitly set the backgroundColor if the window is transparent
+        if (newOptions.transparent) {
+            newOptions.backgroundColor = '#0FFF';
```
Comment (datamadic) [inline src/browser/convert_options.js]: :neckbeard: magic color, with the leading `0` it would be good if the name captured that this is `argb`.

**Comment by HarsimranSingh** [comment]
Running tests now...

**Comment by HarsimranSingh** [comment]
[Win 7 Tests](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/59df8707af26a25be2ffc8cc) (timeout failure passed in isolation)
[Win 10 Tests](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/59df875eaf26a25be2ffc8cd) (timeout failure passed in isolation)

**Pull request merged**